app/views/shared/_contact_panels.html.erb

```diff
@@ -3,179 +3,211 @@
+                    <%= '*' if translation?('contact_panels.chat.smallprint') %>
+                </p>
+
+                <h3 class="contact-panel__opening-times-heading contact-panel-theme-white">Open:</h3>
+                <ul class="t-chat-opening-times contact-panel__list unstyled-list contact-panel-theme-white">
+                    <% unless alternate_hours_active? %>
+                    <li class="contact-panel__additional-info"><%= @footer.web_chat.additional_one %></li>
+                    <li class="contact-panel__additional-info"><%= @footer.web_chat.additional_two %></li>
+                    <li class="contact-panel__additional-info"><%= @footer.web_chat.additional_three %></li>
+                    <% end %>
+                    <% if display_chat_migration_message? %>
+                      <li class="contact-panel__additional-info"><%= t('contact_panels.chat.offline_migration_html') %></li>
+                    <% end %>
+                </ul>
+
+                <% if chat_opening_hours.status_as_string == 'open' %>
+                    <div class="contact-panel__button-container">
+                        <a class="contact-panel__button button t-whatsapp-button" href="https://wa.me/447701342744">
+                          <%= t('contact_panels.chat.available.call_to_action') %>
+                        </a>
+                    </div>
+                <% else %>
+                    <p class="contact-panel-theme-white t-whatsapp-unavailable"><%= t('contact_panels.chat.unavailable.call_to_action') %></p>
+                <% end %>
+
+                <p class="smallprint t-welsh-smallprint"><%= @footer.web_chat.small_print %></p>
+            </div>
+        </div>
+        <div class="l-contact-panel">
+            <div class="contact-panel contact-panel--last">
+                <%= heading_tag level: 2, class: 'contact-panel__heading t-contact-heading' do %>
+                <span class="icon icon--call-us"></span><%= @footer.contact.heading %>
+                <% end %>
+
+                <p class="t-contact-introduction contact-panel__contact-introduction"><%= @footer.contact.introduction %></p>
+
+                <p class="contact-panel__contact-number">
+                    <a class="contact-panel__telephone t-contact-number" href="<%= t('contact.telephone_number_href') %>">
+                        <%= @footer.contact.phone_number %></a>
+                </p>
+
+                <h3 class="contact-panel__opening-times-heading">Open:</h3>
+
+                <ul class="contact-panel__list">
+                    <li class="contact-panel__additional-info"><%= @footer.contact.additional_one %></li>
+                    <li class="contact-panel__additional-info"><%= @footer.contact.additional_two %></li>
+                    <li class="contact-panel__additional-info"><%= @footer.contact.additional_three %></li>
+                </ul>
+                <p class="smallprint"><%= @footer.contact.small_print %></p>
```

Comment:
The content for this is currently missing (not as a result of this work). It appears to need ```t('contact_panels.chat.smallprint')``` here to pull in the text.